editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -1564,6 +1572,21 @@ VisualShaderEditor::VisualShaderEditor() {
 	add_options.push_back(AddOption("ScalarFunc", "Scalar", "Common", "VisualShaderNodeScalarFunc", TTR("Scalar function."), -1, VisualShaderNode::PORT_TYPE_SCALAR));
 	add_options.push_back(AddOption("ScalarOp", "Scalar", "Common", "VisualShaderNodeScalarOp", TTR("Scalar operator."), -1, VisualShaderNode::PORT_TYPE_SCALAR));
 
+	//CONSTANTS
+
+	add_options.push_back(AddOption("E", "Scalar", "Constants", "VisualShaderNodeScalarConstant", TTR("e constant(2.718282). Represents the base of the natural logarithm."), -1, VisualShaderNode::PORT_TYPE_SCALAR, -1, -1, 2.718282f));
```

Comment:
Please put a space before parentheses, so `e constant (2.718282)`.

Reply:
It would be better to use the value from `core/math/math_defs.h` instead of rehardcoding them with lower precision though.